src/controllers/UserController.js

```diff
@@ -1,5 +1,5 @@
 import dotenv from 'dotenv';
-import { User } from '../queries';
+import { User, getAll, create } from '../queries';
```

Comment:
this should be something like `getAllRatings`, use the object approach. Have an object ReadingStat and inside that have all the methods you want. That way you can name them the way you have it above. `getAll` and `Create`. This makes it easier to read and use. You will import the `ReadingStat` object and anywhere you want to use it you will invoke `ReadingStat.create` or `ReadingStat.getAll`

Reply:
ok